dev/com.ibm.ws.classloading.bells/resources/com/ibm/ws/classloading/bells/internal/resources/BellConfigurationMessages.nlsprops

```diff
@@ -79,3 +79,21 @@ bell.error.ctor.useraction=If an exception occurred in the constructor, review a
 bell.no.services.config=CWWKL0058W: Cannot find the {0} service in META-INF/services folder from the {1} library.
 bell.no.services.config.explanation=There is no matching service declared in the META-INF/services folder.
 bell.no.services.config.useraction= No user action is required.
+
+# BETA
+# {0} - library identifier
+bell.spi.visibility.enabled=CWWKL0059I: BELL SPI visibility is enabled for the {0} library. The BELL references the {0} library through a unique class loader that can see SPI packages.
+bell.spi.visibility.enabled.explanation=To prevent applications and resources from seeing SPI packages, the BELL references the library through a unique class loader that is not shared with applications that reference the same library.
+bell.spi.visibility.enabled.useraction=Disable SPI visibility in the BELL configuration whenever the BELL must reference a library through a class loader that is shared with applications that reference the same library. Otherwise, no action is required.
+
+# BETA
+bell.spi.visibility.disabled.libref.global=CWWKL0060W: BELL SPI visibility is not enabled for the Liberty global shared library.
```

Comment:
The message and the explanation seem to be saying almost, but not quite, the same thing,  in a way that might be confusing, particularly in translation. Is it possible to have the initial message identify the problem that is caused- ie the BELL configuration is referencing the global shared library? Something like

```suggestion
bell.spi.visibility.disabled.libref.global=CWWKL0060W: The BELL configuration must not reference the Liberty global shared library.
```

Reply:
@dmuelle Thanks for the review.  I'll address all these comments in my next PR for this EPIC, which will contain changes for the same messages file.